src/static/js/canto_dex.js

```diff
@@ -0,0 +1,113 @@
+
+const ComptrollerAddress = '0x5e23dc409fc2f832f83cec191e245a191a4bcc5c'
+
+const ctokenAddresses = [
+    "0xd6a97e43fc885a83e97d599796458a331e580800", // NOTE-USDC
+    "0xf0cd6b5ce8a01d1b81f1d8b76643866c5816b49f", // NOTE-USDT
+    "0x3c96dcfd875253a37acb3d2b102b6f328349b16b", // NOTE-wCANTO
+    "0xb49a395b39a0b410675406bee7bd06330cb503e3", // ETH-wCANTO
+    "0xc0d6574b2fe71eed8cd305df0da2323237322557", // wCANTO-ATOM
+]
+
+const decimals = {
+    "0x4e71a2e537b7f9d9413d3991d37958c0b5e1e503": 18, // NOTE
+    "0x826551890dc65655a0aceca109ab11abdbd7a07b": 18, // CANTO
+    "0x5fd55a1b9fc24967c4db09c513c3ba0dfa7ff687": 18, // WETH
+    "0xeceeefcee421d8062ef8d6b4d814efe4dc898265": 6, // ATOM
+    "0x7264610a66eca758a8ce95cf11ff5741e1fd0455": 18, //CINU
+    "0x80b5a32e4f032b2a058b4f29ec95eefeeb87adcd": 6, // USDC
+    "0xd567b3d7b8fe3c79a1ad8da978812cfc4fa05e75": 6, //USDT
+    "0xd6a97e43fc885a83e97d599796458a331e580800": 18, // NOTE-USDC
+    "0xf0cd6b5ce8a01d1b81f1d8b76643866c5816b49f": 18, // NOTE-USDT
+    "0x3c96dcfd875253a37acb3d2b102b6f328349b16b": 18, // NOTE-wCANTO
+    "0xb49a395b39a0b410675406bee7bd06330cb503e3": 18, // ETH-wCANTO
+    "0xc0d6574b2fe71eed8cd305df0da2323237322557": 18, // wCANTO-ATOM
+}
+
+const cantoAddress = "0x826551890dc65655a0aceca109ab11abdbd7a07b"
+
+const pairNames = [
+    ["NOTE", "USDC"],
+    ["NOTE", "USDT"],
+    ["NOTE", "wCANTO"],
+    ["ETH", "wCANTO"],
+    ["wCANTO", "ATOM"],
+]
+
+async function main() {
+    const App = await init_ethers();
+
+    _print(`Initialized ${App.YOUR_ADDRESS}\n`);
+    _print("Reading smart contracts...\n");
+
+    const prices = await getCantoPrices();
+    for (let k in prices) {
+        prices[k.toLowerCase()] = prices[k];
+    }
+
+    const rewardTokenTicker = "WCANTO";
+    const poolCount = ctokenAddresses.length;
+    const BaseV1Router = new ethers.Contract("0xa252eee9bde830ca4793f054b506587027825a8e", BaseV1Router_ABI, App.provider);
+    const Comptroller = new ethers.Contract(ComptrollerAddress, Comptroller_ABI, App.provider);
+
+    for (i = 0; i < poolCount; i++) {
+        var Ctoken = new ethers.Contract(ctokenAddresses[i], CTOKEN_ABI, App.provider);
```

Comment:
This CTOKEN_ABI variable is not defined anywhere? are you sure this is working?

Also change the variable name

Reply:
CTOKEN_ABI is already defined in the src/static/data.js of this project